### DISCUSSION TITLE
Alpha 145 - The Big Profiles Update

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4156] [Release Notes]
## New profile settings

The new "Profiles" settings tab unifies all profile types in a single settings UI:

![](https://i.imgur.com/BUnC089.png)

* Groups, hotkeys, colors, tab icons, title settings and login scripts are now available for all profile types:

![](https://i.imgur.com/eOj1XW5.png)

* ⚠️ You'll have to re-select your default profile and re-set profile hotkeys if you had any.
* The "Built-in" profiles (every standard shell and serial port) can be optionally hidden if you're only using your own custom profiles.
* The profile selector now shows all profiles, but there are still separate hotkeys to show SSH, serial and telnet profile selectors.
* You can omit the baud rate for serial profiles if you want Tabby to ask for baud rate every time when opening a serial tab.
* Added automatic *Developer Tools for Visual Studio* profiles on Windows.

## Socket / Telnet client

* Added a plain socket ("Telnet") profile type. Telnet profiles have the same input modes as serial profiles, and default to a readline input (same as `nc`):

![](https://i.imgur.com/QatmYVg.png)

## Miscellaneous
* Updated the connection toolbar design - the toolbar is now pinnable and is easier to discover.
* Setting `%COMSPEC%` on Windows is now optional and off by default.

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.145'>Alpha 145 - The Big Profiles Update</a>.</em>

**Comment by lizhibo0509** [comment]
I encountered some serious problems when I upgraded tabby-1.0.145. start the program，I can not found anything like setting button、host button，just blank. so I have to rollback  tabby-1.0.144

OS: windows
Version: 20H2  19042.1083
